release_bot/releasebot.py

```diff
@@ -68,13 +69,19 @@ def make_new_pypi_release(self):
                                                                 'release-conf.yaml'))
         self.new_release.update(release_conf)
         self.pypi.release(self.new_release)
+        msg = f"Released {self.new_release['version']} on PyPI"
+        self.logger.info(msg)
+        self.github.add_comment(self.new_release['pr_id'], msg)
         return True
 
     def make_new_fedora_release(self):
         if self.new_release['fedora']:
             self.logger.info("Triggering Fedora release")
             self.fedora.release(self.new_release)
             self.new_release['tempdir'].cleanup()
+            msg = "Finished Fedora release"
```

Comment:
Could we also get Fedora version here?

Reply:
Just the branches which we successfully updated / built from.
To get exact N-V-R I'd probably need to parse `fedpkg build` output.